src/CoreServices/FSEvents.cs

```diff
@@ -199,6 +199,43 @@ public FSEventStream (string [] pathsToWatch, TimeSpan latency, FSEventStreamCre
 		{
 		}
 
+		[DllImport (Constants.CoreServicesLibrary)]
+		static extern IntPtr FSEventStreamCreateRelativeToDevice (IntPtr allocator, FSEventStreamCallback callback, 
+			IntPtr context, ulong deviceToWatch, IntPtr pathsToWatchRelativeToDevice, 
+			ulong sinceWhen, double latency, FSEventStreamCreateFlags flags);
+
+		private void FSEventStreamInit (CFAllocator allocator, ulong deviceToWatch, NSArray pathsToWatchRelativeToDevice, 
```

Comment:
Style: we usually don't use `private`, it's redundant.

```suggestion
	void FSEventStreamInit (CFAllocator allocator, ulong deviceToWatch, NSArray pathsToWatchRelativeToDevice, 
```